sourced/ml/mining/__main__.py

```diff
@@ -0,0 +1,56 @@
+import argparse
+import logging
+import sys
+
+from modelforge import slogging
+
+from .cmd import ArgumentDefaultsHelpFormatterNoNone
+
+
+def get_parser() -> argparse.ArgumentParser:
```

Comment:
```suggestion
def parse_args() -> argparse.Arguments:  or whatever the type for the returned parsed object
```

Reply:
Done

Reply:
as below, can't do this or I cam't display the help in the main